src/PowerShellEditorServices/Services/PowerShell/Host/PsesInternalHost.cs

```diff
@@ -527,6 +533,18 @@ public IReadOnlyList<TResult> InvokePSCommand<TResult>(PSCommand psCommand, Powe
             return task.ExecuteAndGetResult(cancellationToken);
         }
 
+        void IInternalPowerShellExecutionService.UnsafeInvokePSCommand(
+            PSCommand psCommand,
+            PowerShellExecutionOptions executionOptions,
+            CancellationToken cancellationToken)
+            => InvokePSCommand<PSObject>(psCommand, executionOptions, cancellationToken);
+
+        IReadOnlyList<TResult> IInternalPowerShellExecutionService.UnsafeInvokePSCommand<TResult>(
+            PSCommand psCommand,
+            PowerShellExecutionOptions executionOptions,
+            CancellationToken cancellationToken)
+            => InvokePSCommand<TResult>(psCommand, executionOptions, cancellationToken);
+
         public void InvokePSCommand(PSCommand psCommand, PowerShellExecutionOptions executionOptions, CancellationToken cancellationToken) => InvokePSCommand<PSObject>(psCommand, executionOptions, cancellationToken);
```

Comment:
Perhaps we should make this private and _require_ the `Unsafe` versions be called?